### DISCUSSION TITLE
SNHU and my success story for getting accepted into GitHub Education

### DISCUSSION THREAD

**Discussion by Xiodrade** [community/community#52837] [GitHub Education]
For anyone with SNHU and online - here's my success story with joining. I submitted 10 requests and on the 10th I was finally accepted.

The first 9 requests - It's important to note that during these requests I was using a webcam and taking pictures of the PC screen... which we all know the quality of that.
These requests range from taking pictures of my academic profile to my unofficial transcript and even my financial aid offer letter.
My academic profile picture showed my student ID, the school name, my current registered classes along with the dates for those, my full name which matches my github profile, AND my home address which matches the location I've been providing. 

I have a school ID, but I am an online student. It does not have the date. 

The 10th request and what got me finally accepted...
I requested my official transcript to be electronically sent to me. This transcript contained the SNHU logo at the bottom, the current date on the top right, and all my personal information on the top left which included my name, my address, and my student ID.

Once I received this I printed it out on paper, stuck it to my white board, and used my phones camera to take a picture. It took a while to ensure all data was there in one picture such as the logo, my name, and the current date, but once I got the perfect picture I was surprised to see I was finally accepted!

**Comment by samus-aran** [comment]
Thanks @Xiodrade for sharing your process 👏  I'm sure that will help others in similar situations 👍

**Comment by chaseleinart** [comment]
I was also able to get accepted after 6 attempts as an online SNHU student. I requested a copy of my transcript and took clear photos with name, dates, logo, and everything visible, but no dice. I tried other methods, but the only thing that worked was requesting a physical student ID and taking a clear picture of it
